.github/workflows/ci.yml

```diff
@@ -528,6 +528,9 @@ jobs:
       matrix:
         box:
           - fedora/39-cloud-base
+          # the upstream rockylnux/8 image no longer works:
```

Comment:
If we're keping Rocky Linux 8 to test compatibility, can you add a comment to the file so it doesn't get changed again?

Reply:
Should we just remove the rockylinux/9 one as well?

I notice in Vagrant Cloud images can be hosted externally, but then the link dies after a year or two and the image is unusable. This is also happening with the fedora 37 images. If we use one should we just use all the bento images since they are hosted by vagrant cloud?

Reply:
> rockylinux/9

Probably safe to retain

> Vagrant Cloud

Maybe we can just switch away from Vagrant to Lima
https://github.com/Mirantis/cri-dockerd/commit/a3a2995e3fab6720b660d65c8c18689fec426eb0